src/alignment/sparse.rs

```diff
@@ -90,15 +90,13 @@ pub fn lcskpp(matches: &[(u32, u32)], k: usize) -> SparseAlignmentResult {
         n = max(n, x + k);
         n = max(n, y + k);
     }
-    events.sort();
+    events.sort_unstable();
```

Comment:
I hope using `sort_unstable` over `sort` doesn't change any behaviour, but then again, we have tests so I do not have to "hope" at all ;)

Reply:
Clippy suggest using `sort_unstable()` because the value in the vector has primitive types, `(u32, u32, u32)`. `sort_unstable()` differs from `sort()` by not ensuring preserving the order of equal elements. 

In this case, for example,  `[(5,5,5), (5,5,5), (5,5,6)]` is certainly the same as `[(5,5,5), (5,5,5), (5,5,6)]` so this will not change the behaviour, while improving performance.